include/onnxruntime/core/platform/EigenNonBlockingThreadPool.h

```diff
@@ -130,6 +130,26 @@
 namespace onnxruntime {
 namespace concurrency {
 
+class Profiler {
+ public:
+  Profiler() = default;
+  ~Profiler() = default;
+  ORT_DISALLOW_COPY_ASSIGNMENT_AND_MOVE(Profiler);
+  using CLOCK = std::chrono::high_resolution_clock;
```

Comment:
GetStartTimeNs() returns the time point when default Profiler starts, how we supposed to consume that since the thread pool profiler is supposed to capture its own starts for different section?